scripts/deploy/staking-pool-deploy-no-etherscan.ts

```diff
@@ -0,0 +1,16 @@
+import {StakingPoolMediator, StakingPoolFactory} from '../../typechain-types'
+import {deployContract, deployContractWithProxy} from '../utils/contract'
+
+export async function deployStakingPool(
+    tokenSweepBeneficiary: string
+): Promise<void> {
+    const factory = await deployContract<StakingPoolFactory>(
```

Comment:
atm these scripts wont work, since the StakingPoolFactory doesnt have an constructor defined - the `tokenSweepBeneficiary` needs to be initialized via the initializer.